src/app/components/ConsentBanner/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { string, element } from 'prop-types';
+import styled from 'styled-components';
+import { FF_NEWS_SANS_REG } from '@bbc/psammead-styles/fonts';
+import {
+  GEL_GREAT_PRIMER,
+  GEL_LONG_PRIMER,
+} from '@bbc/gel-foundations/typography';
+import {
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_MARGIN_BELOW_400PX,
+  GEL_MARGIN_ABOVE_400PX,
+} from '@bbc/gel-foundations/spacings';
+
+const GREY = '#323232';
+const ORANGE = '#fa9900';
+const WHITE = '#ffffff';
+const LIGHTGREY = '#bdbdbd';
+
+const Wrapper = styled.div`
+  font-family: ${FF_NEWS_SANS_REG};
+  background-color: ${GREY};
+  padding: 16px ${GEL_MARGIN_BELOW_400PX};
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: ${GEL_MARGIN_ABOVE_400PX};
+  }
+`;
+
+const CenterWrapper = styled.div`
+  max-width: 1280px;
+  margin: 0 auto;
+
+  &::after {
```

Comment:
This is here to ensure that the background colour covers the banner as the inner elements are float. The alernative is to have another div inside, but this is just copied from orbit.

Reply:
done :)